src/main/java/net/minecraftforge/common/BiomeDictionary.java

```diff
@@ -25,6 +26,11 @@
 
 import net.minecraft.world.level.biome.Biome;
 
+/**
+ * Use {@link Tags.Biomes} when possible.
+ * Note that tags are not usable yet during {@link BiomeLoadingEvent} so BiomeDictionary are still necessary there
```

Comment:
Grammatical error: `are` to `is`.
```suggestion
 * Note that tags are not usable yet during {@link BiomeLoadingEvent} so BiomeDictionary is still necessary there
```